RealmModelGenerator.xcodeproj/xcuserdata/zz.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this file shouldn't be tracked by git. if you look at the path. .xcodeproj/xcuserdata/
any use data file should not be tracked by git